solutions/remotemonitoring/scripts/individual/deployment-configmap.yaml

```diff
@@ -25,49 +27,21 @@ data:
   # and resource type "Iot Hub" then "Shared access policies" menu.
   iothub.connstring: "{IoT Hub connection string}"
 
-
   # The connection string Document DB will be of format "AccountEndpoint={URI};AccountKey={Key};"
   # The key will be shown in CLI window as output when remote-cli command has finished running.
   # Alternatively it can be accessed from http://portal.azure.com under {myResourceGroupName}
   # and resource type "Azure Cosmos DB account" then "Keys" menu.
   docdb.connstring: "{DocumentDB connection string}"
 
-  azureblob.account: "{Azure Blob account name}"
-  azureblob.key: "{Azure Blob account key}"
-  azureblob.endpointsuffix: "{Azure Blob account endpoint suffix}"
   azureblob.connstring: "{Azure Blob connection string}"
 
-  azure.resourcemanager.endpoint.url: "{Azure Resource Manager endpoint url}"
-
-  # Authentication
-  auth.webservice.url: "http://auth-svc:9001/v1"
-
   # Storage Adapter
   storageadapter.webservice.url: "http://storage-adapter-svc:9022/v1"
 
-  # IoT Hub Manager
-  iothubmanager.webservice.url: "http://iothub-manager-svc:9002/v1"
-
   # Device Simulation
-  devicesimulation.webservice.url: "http://device-simulation-svc:9003/v1"
   devicesimulation.template: "remote-monitoring-simulations-template"
```

Comment:
This doesn't look being referenced anywhere.

Reply:
It is used by device-simulation.yaml at least in this branch https://github.com/Azure/pcs-cli/blob/keyvault-br-2/solutions/remotemonitoring/scripts/individual/device-simulation.yaml. 

They are both removed in the the 3rd PR with the extra cleanup though.